src/deprecated/support.js

```diff
@@ -0,0 +1,11 @@
+define( [
+	"../var/support"
+], function( support ) {
+
+"use strict";
+
+support.trim = "" === "\uFEFF\xA0".trim();
```

Comment:
Nice optimization! I checked and it still fails on Android 4.0 as expected.

Reply:
To be clear, I am arguing against the introduction of `support.trim` at +29 bytes (and really, even at +10).